src/vivarium_cluster_tools/psimulate/results/cli_options.py

```diff
@@ -18,3 +18,13 @@
     help="Hidden developer option, if flagged, don't automatically "
     "cleanup results directory on failure.",
 )
+with_sim_backup = click.option(
+    "--make-backups", is_flag=True, help="Periodically save simulation state to disk."
+)
+backup_freq = click.option(
+    "--backup-freq",
+    type=int,
+    default=5,
+    show_default=True,
+    help="Interval in hours between saving backups.",
```

Comment:
Maybe add to the help string that it is only applicable if `--make-backups`.

Actually...do you need both? Could you just get by w/ `--backup-freq` (where setting to zero would be the default and effectively like the current default of `--make-backups` being false?

Reply:
Yeah, i went back and forth a bit about this because I'd prefer to add fewer flags all things being equal but it also seems good to be more explicit

Reply:
I think if we want backups to be the default behavior we only need the frequency flag and no backups would be 0. I'd also prefer less flags and I think you could set a bool trigger for logic with the input from frequency so you shouldn't have to change that much.

Reply:
done